ui/hooks/src/use-reflections.ts

```diff
@@ -88,26 +92,24 @@ export const useReflections = (props: UseReflectionProps) => {
 
   const [errors, setErrors] = React.useState<(ApolloError | Error)[]>([]);
 
-  const getterHook =
-    entityType === EntityTypes.BEAM
-      ? useGetReflectionsFromBeamLazyQuery
-      : useGetReflectReflectionsLazyQuery;
+  const indexingDID = React.useRef(getSDK().services.gql.indexingDID);
 
-  const mergedVars: GetReflectionsFromBeamQueryVariables = React.useMemo(() => {
-    const vars: GetReflectionsFromBeamQueryVariables = {
-      id: entityId,
+  const mergedVars: GetReflectionStreamQueryVariables = React.useMemo(() => {
+    const vars: GetReflectionStreamQueryVariables = {
+      indexer: indexingDID.current,
       sorting: { ...defaultSorting, ...(sorting ?? {}) },
     };
     if (filters) {
       vars.filters = filters;
     }
     return vars;
-  }, [entityId, sorting, filters]);
+  }, [sorting, filters]);
 
-  const [fetchReflections, reflectionsQuery] = getterHook({
+  const [fetchReflections, reflectionsQuery] = useGetReflectionStreamLazyQuery({
     variables: {
       ...mergedVars,
-      first: overscan,
+      last: overscan,
```

Comment:
Is this actually `last` or indeed `first` ? Did you test it ?

Reply:
I tested both and if I use a high overscan number (120), it doesn't really make any difference if it's `first` or `last` and it still displays fine. But if I use a low `overscan` number, it will shift the reflections no matter whether it's `first` or `last`.

Reply:
this is tricky as the overscan defines how many items to fetch at once, having it set to a high number will only hide the problem (aka overscan=120 for a small list it basically means fetch/refetch everything) and it will bite us in the future.

Reply:
I see that the `estimatedHeight` is also set to 120 when being passed to the Virtual List. Does this number mean something related to the estimated number of items?

Reply:
no, that is the estimated height of a card - this estimation is used before measuring the actual height of the card

Reply:
Thanks for explaining that :). Now I'm testing `overscan` with 25